.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+dist
```

Comment:
**[OPTIONAL]**
It's really good that you added `dist` in the ignore, but we are using `CRA` or create-react-app. So, we also need to add `build` folder in here.